crypto/asn1/a_gentm.c

```diff
@@ -157,10 +157,10 @@ int ASN1_GENERALIZEDTIME_print(BIO *bp, const ASN1_GENERALIZEDTIME *tm)
     if (BIO_printf(bp, "%s %2d %02d:%02d:%02d%.*s %d%s",
                    _asn1_mon[M - 1], d, h, m, s, f_len, f, y,
```

Comment:
If not talk generically, one other method to deal with this `_asn1_mon` thing is to 'squash' the two functions into one to reduce duplicated code (as what is done for the `asn1_[gen/utc]_to_tm` functions)